docs/app/Examples/modules/Accordion/Types/Accordion.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react'
 import { Accordion, Icon } from 'stardust'
 
-export default class AccordionStyledExample extends Component {
+export default class AccordionAccordionExample extends Component {
```

Comment:
No, just one of our ugly docs patterns.

```
${componentName}${exampleName}Example.js
```

So you get `AccrdionStyledExample.js` for instance.  But the first example name in SUI turns out to just be the component name, so we have `ButtonButtonExample.js` etc.  Docs are set to be overhauled and cleaned up once components are done.